### DISCUSSION TITLE
DTU von Hoymiles und Open DTU

### DISCUSSION THREAD

**Discussion by chkbln** [tbnobody/OpenDTU#1450] [General]
Ich habe mir die Open DTU nachgebaut (WR Hoymiles HMS1600 +ESP32 +cmt9300A). Aber ich bekomme keine Antwort vom Wechelsrichter. Dann habe ich  ESP32 und CMT9300a getauscht. Anderes PIN Mapping ausprobiert. Nix hat geholfen. Heute habe ich dann meine Hoymiles DTU vom Strom getrennt. Und siehe da Open DTU funktioniert. Ist es richtig, dass beide DTU´s nicht parallel miteinander funkionieren? Wenn das so ist, sollte das meiner Meinung nach  hier beschrieben werden. 
Gruß

**Comment by tbnobody** [comment]
Die Frage ist immer wo $Menschen nach Infos suchen.... https://github.com/tbnobody/OpenDTU/wiki/FAQ#q-opendtu-shows-wrongunrealistic-or-just-missing-values
